express-post-json/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const app = express();
+
+let nextID = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const key in grades) {
+    gradesArray.push(grades[key]);
+  }
+  if (!grades) {
+    res.status(404).send('No grades :(');
+  } else {
+    res.status(200).json(gradesArray);
+  }
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades[nextID] = req.body;
+  grades[nextID].id = nextID;
+  grades[nextID].score = parseInt(grades[nextID].score, 10);
+  nextID++;
+  res.status(201).send(grades);
```

Comment:
The client should only receive the newest object in the response body, rather than the entire data model.